src/tasks.py

```diff
@@ -39,7 +39,7 @@ def ami_for_project(pname):
     
     print len(results),"results"
 
-    utils.table(results, ['id', 'root_device_type', 'virtualization_type', 'name'])
+    print utils.table(results, ['id', 'root_device_type', 'virtualization_type', 'name'])
```

Comment:
there is a decorator called `echo_output` that prints the result of the task. I prefer using that to printing reports of things, especially if a task can be called by itself or as part of another function (like this one could)